### PR TITLE
CTW-473/status refresh

### PR DESCRIPTION
**Pull request by pmadria3**
CTW-473

This PR adds status message and Last Retrieved date (if there is one) to the other providers table. 
Screenshot 2023-02-01 at 12 44 32 PM

### CODE REVIEW

**Review by mrosata — APPROVED** [review]
lgtm

File: src/components/content/conditions.tsx [hunk 1]
```diff
@@ -287,17 +288,32 @@ export const Conditions = withErrorBoundary(
             />
           </div>
           <div className="ctw-space-y-3">
+            {(patientHistoryInfo?.status === "in_progress" ||
+              patientHistoryInfo?.status === "initialize") && (
+              <PatientHistoryStatus
+                status={patientHistoryInfo.status}
+                date={patientHistoryInfo.dateCreated}
+              />
+            )}
```
Comment (WesleyKapow) [inline src/components/content/conditions.tsx:297]: Why is this guard needed, won't `PatientHistoryStatus` take care of rendering or not rendering something for us?

Comment (pmadria3) [reply src/components/content/conditions.tsx]: ah yes you are right that was the whole point of the switch lol

File: src/components/content/conditions.tsx [hunk 2]
```diff
@@ -293,33 +293,32 @@ export const Conditions = withErrorBoundary(
...
-                  )}
+              {shouldShowClinicalHistoryArea &&
+                !readOnly &&
+                !hideRequestRecords && (
+                  <div className="ctw-flex ctw-items-baseline ctw-space-x-2">
+                    {patientHistoryInfo?.lastRetrievedAt && (
+                      <div className="ctw-text-sm ctw-italic ctw-text-black">
+                        Last Retrieved {patientHistoryInfo.lastRetrievedAt}
+                      </div>
+                    )}
```
Comment (WesleyKapow) [inline src/components/content/conditions.tsx:310]: Eh this isn't quite right, we can show "Last Retrieved" even if it is read only and even if we are hiding the request records link.

**Review by WesleyKapow — CHANGES_REQUESTED** [review]

File: src/components/content/conditions.tsx (hunk 2 above)
Comment (pmadria3) [reply src/components/content/conditions.tsx]: Ah good catch - moved it outside

**Review by luthfib — APPROVED** [review]

**Review by WesleyKapow — APPROVED** [review]

**Pull request merged**